resources/aws/elb.go

```diff
@@ -0,0 +1,159 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
+	awsclient "github.com/giantswarm/aws-operator/client/aws"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+// ELB is an Elastic Load Balancer
+type ELB struct {
+	Name             string
+	dnsName          string
+	hostedZoneID     string
+	AZ               string
+	SecurityGroup    string
+	Tags             []string
+	Client           *elb.ELB
+	LoadBalancerPort int
+	InstancePort     int
+}
+
+func (lb *ELB) CreateIfNotExists() (bool, error) {
+	if lb.Client == nil {
+		return false, microerror.MaskAny(clientNotInitializedError)
+	}
+
+	if err := lb.CreateOrFail(); err != nil {
+		if err.Error() == awsclient.ELBAlreadyExists {
```

Comment:
You will need to use `strings.Contains` here - this error was masked before, so its message will be not exactly equal to the message defined in the global variable.